src/Config.cpp

```diff
@@ -105,6 +109,17 @@ void Config::Load(json config)
 		}
 	}
 
+	// Process `--websocket_host` override
+	QString host_argument = Utils::Platform::GetCommandLineArgument(CMDLINE_WEBSOCKET_HOST);
+	if (host_argument != "") {
+		blog(LOG_INFO, "[Config::Load] --websocket_host passed. Overriding WebSocket host with: %s",
+		     host_argument.toStdString().c_str());
+		ServerHost = host_argument.toStdString();
+		HostOverridden = true;
+	}
+
+
+
 	// Process `--websocket_ipv4_only` override
 	if (Utils::Platform::GetCommandLineFlagSet(CMDLINE_WEBSOCKET_IPV4_ONLY)) {
```

Comment:
This should probably log a warning if the address is overridden, and ensure that the `Ipv4Only` variable is not set to true.